app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Managing behaviour in the classroom"
+heading: "Managing behaviour in the classroom"
+subcategory: Qualifications and experience
+description: |-
+  Teacher training adviser Jane Wilkinson answers questions about how to manage behaviour in the classroom.
+date: "2024-01-09"
+backlink: "../../"
+navigation: 5.35
+navigation_title: How can I manage behaviour in the classroom?
+navigation_description: Teacher training adviser Jane Wilkinson answers common concerns around behaviour management.
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+related_content:
+    How to become a teacher : "/steps-to-become-a-teacher"
+    What to expect in teacher training : "/train-to-be-a-teacher/initial-teacher-training" 
+    Early career teachers (ECTs) : "/support-for-early-career-teachers"
+
+keywords:
+  - Behaviour management
+  - Managing behaviour
+  - Disruptive behaviour
+  - Challenging behaviour
+  - Disruptive pupils
+  - Difficult pupils
+  - Positive behaviour
+  - Classroom teaching
+  - NQT
+  - National professional qualification
+  - Support
+  - Training
+  - Teacher training
+  - ECF
+  - Teaching
+  - How to become a teacher
+  - Teacher training courses
+  - Behaviour
+  - Management
+  - classroom concerns
+---
+
+Teacher training adviser Jane Wilkinson has years of teaching experience. She answers some common concerns that trainee teachers have around how to manage behaviour in the classroom.
+
+## How can I create a positive learning environment?
+In my experience of teaching hundreds of thousands of children, if they’re engrossed in their learning and know what it is they need to be doing, they'll be much less disruptive. The biggest challenge isn't usually big things it’s more the chatting and low-level disruption.  
+
+Make sure you’re very clear. If children understand your expectations of them in terms of workload, behaviour and outcome and you scaffold that journey then behaviour becomes much less of a problem. Start off by modelling what you would want in your classroom. Do you want the children lining up at the start of the lesson? Or do you want them coming straight in and starting with the work?
```

Comment:
```suggestion
Make sure you’re very clear. If children understand your expectations of them in terms of workload, behaviour and outcomes and you scaffold that journey then behaviour becomes much less of a problem. Start off by modelling what you would want in your classroom. Do you want the children lining up at the start of the lesson? Or do you want them coming straight in and starting with the work?
```

also - is there an easier way of saying 'scaffold that journey'?

Reply:
@gemmadallmandfe I've just deleted scaffold that journey as I don't think it's needed.